src/components/Notification/Notification.tsx

```diff
@@ -2,43 +2,68 @@ import * as moment from "moment";
 import * as React from "react";
 import * as css from "./Notification.scss";
 import { IOperation, OperationsStatus } from "reducers/operations";
+import classNames = require("classnames");
 
 interface IProps {
   operation: IOperation;
 
   close: () => any;
 }
 
-export default class Notification extends React.Component<IProps, null> {
+export default class Notification extends React.Component<IProps, { minimized: boolean }> {
```

Comment:
lets create an IState interface for the state

Reply:
updated: https://github.com/daostack/alchemy/pull/52/commits/191f7d3b870fba072c530d61301349f4f3f7ff94